examples/light-switch-app/nrfconnect/main/BindingHandler.cpp

```diff
@@ -0,0 +1,304 @@
+    {
+    case Clusters::OnOff::Commands::Toggle::Id:
+        Clusters::OnOff::Commands::Toggle::Type toggleCommand;
+        ret = Controller::InvokeCommandRequest(device->GetExchangeManager(), device->GetSecureSession().Value(), binding.remote,
+                                               toggleCommand, onSuccess, onFailure);
+        break;
+
+    case Clusters::OnOff::Commands::On::Id:
+        Clusters::OnOff::Commands::On::Type onCommand;
+        ret = Controller::InvokeCommandRequest(device->GetExchangeManager(), device->GetSecureSession().Value(), binding.remote,
+                                               onCommand, onSuccess, onFailure);
+        break;
+
+    case Clusters::OnOff::Commands::Off::Id:
+        Clusters::OnOff::Commands::Off::Type offCommand;
+        ret = Controller::InvokeCommandRequest(device->GetExchangeManager(), device->GetSecureSession().Value(), binding.remote,
+                                               offCommand, onSuccess, onFailure);
+        break;
+    default:
+        LOG_DBG("Invalid binding command data - commandId is not supported");
+        break;
+    }
+    if (CHIP_NO_ERROR != ret)
+    {
+        LOG_INF("Invoke Unicast Command Request ERROR: %s", ErrorStr(ret));
+    }
+}
+
+void BindingHandler::OnOffProcessCommandGroup(CommandId commandId, const EmberBindingTableEntry & binding, void * context)
+{
+    LOG_INF("Group OnOff Process Binding command...");
+    auto sourceNodeId = Server::GetInstance().GetFabricTable().FindFabricWithIndex(binding.fabricIndex)->GetNodeId();
+    Messaging::ExchangeManager & exchangeMgr = Server::GetInstance().GetExchangeManager();
+    CHIP_ERROR ret                           = CHIP_NO_ERROR;
+    switch (commandId)
+    {
+    case Clusters::OnOff::Commands::Toggle::Id:
+        Clusters::OnOff::Commands::Toggle::Type toggleCommand;
+        ret =
+            Controller::InvokeGroupCommandRequest(&exchangeMgr, binding.fabricIndex, binding.groupId, sourceNodeId, toggleCommand);
+        break;
+
+    case Clusters::OnOff::Commands::On::Id:
+        Clusters::OnOff::Commands::On::Type onCommand;
+        ret = Controller::InvokeGroupCommandRequest(&exchangeMgr, binding.fabricIndex, binding.groupId, sourceNodeId, onCommand);
+        break;
+
+    case Clusters::OnOff::Commands::Off::Id:
+        Clusters::OnOff::Commands::Off::Type offCommand;
+        ret = Controller::InvokeGroupCommandRequest(&exchangeMgr, binding.fabricIndex, binding.groupId, sourceNodeId, offCommand);
+        break;
+    }
+    if (CHIP_NO_ERROR != ret)
+    {
+        LOG_INF("Invoke Group Command Request ERROR: %s", ErrorStr(ret));
+    }
+}
+
+void BindingHandler::LevelControlProcessCommandUnicast(CommandId commandId, const EmberBindingTableEntry & binding,
+                                                       DeviceProxy * device, void * context)
+{
+    LOG_DBG("Unicast Level Control Process Binding command {0x%x}-> {%d}...", binding.remote, (uint16_t) commandId);
```

Comment:
nit: The log may not be understandable by people who don't have a detailed knowledge of the code. I think that something like "Sending Level Control command %x to bound node 0x%x" would be enough.